fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -4960,6 +4962,20 @@ public LogicalPlan visitShowBackends(ShowBackendsContext ctx) {
         return new ShowBackendsCommand();
     }
 
+    @Override
+    public LogicalPlan visitShowBackup(ShowBackupContext ctx) {
+        String dbName = null;
+        Expression wildWhere = null;
+        if (ctx.database != null) {
+            List<String> nameParts = visitMultipartIdentifier(ctx.database);
+            dbName = nameParts.get(0);
```

Comment:
why dbName is always `nameParts.get(0)`? how about `catalog.db`? if we do not support `catalog.db`, maybe we should change `multipartIdentifier` to `identifier` in syntax(DorisParser.g4)?